running-automation-testing.md

```diff
@@ -152,23 +153,26 @@ curl -X GET https://api.kobiton.com/v1/sessions
 
 Below are necessary Kobiton Rest API endpoints that you may need.
 
-- Get session info
-```
-GET /sessions{sessionId}
-```
+### Authorization
+
 To make a request:
 ```
 curl -X GET https://api.kobiton.com/v1/sessions/{sessionId}
   -H 'Authorization: Basic dGVzdHVzZXI6MTIzZWQtMTIzZmFjLTkxMzdkY2E='
   -H 'Accept: application/json'
 ```
 
+### Get session info
+```
+GET /sessions{sessionId}
+```
+
 With this line of code, you can print the session information to the console. 
 ```javascript
 console.log(sessionCapabilities)
```

Comment:
`sessionCapabilities` is not defined anywhere in the document. This may confuse the readers
Suggestion: replace this javascript function with the JSON response of the request, or simply remove these lines